libs/sdk-ui-kit/src/ChartSorting/types.ts

```diff
@@ -0,0 +1,128 @@
+// (C) 2022 GoodData Corporation
+import { SortDirection } from "@gooddata/sdk-model";
+import { ISortItem, LocalIdRef, ILocatorItem } from "@gooddata/sdk-model";
+
+/**
+ * @internal
+ */
+export interface ISortConfig {
```

Comment:
Don't we have this type somewhere else and can we just reuse it without redefining it again?

Reply:
To use them will need to add `sdk-ui-ext` as dependency. Not sure if is worth just for this.